app/views/foods/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Search Results<h2>
+<h3>Total Results: <%= number_with_delimiter(@foods[0], :delimiter => ',') %> </h3>
+
+<ol>
+  <% @foods[1].each do |food| %>
```

Comment:
If you use the syntax that I mentioned in the first comment, then you'll be able to send over two variables, which helps with having readable code.